src/cloud/components/ContentManager/ContentManagerToolbar.tsx

```diff
@@ -734,7 +734,7 @@ const Container = styled.div`
 
   .item__due-date__select,
   .item__status__select,
-  .item__assignee__select {
+  .assignee__select {
```

Comment:
Instead of changing the css here, change the class in `Pickers/AssigneeSelect` to keep consistency with other namings

Reply:
Fixed